src/Models/GameModel.php

```diff
@@ -67,12 +67,17 @@ public function addNewGame(
         }
     }
 
-    public function softDeleteGame($id): bool
+    public function toggleGameStatus($id, $deleted): bool
     {
-
-        $query = $this->db->prepare("
+        if ($deleted == 1) {
+            $query = $this->db->prepare("
+        UPDATE `games` SET `games`.`deleted` = 0 WHERE `games`.`id` = :id;
+        ");
+        } else if ($deleted == 0) {
+            $query = $this->db->prepare("
         UPDATE `games` SET `games`.`deleted` = 1 WHERE `games`.`id` = :id;
```

Comment:
This bit you can make DRYer. You can have the query just once as a string, but pass in 1 or 0 depending on what $deleted is set to